src/plone/app/workflowmanager/browser/controlpanel.py

```diff
@@ -312,6 +348,12 @@ def handle_response(self, message=None, tmpl=None, redirect=None,
             if tmpl and not justdoerrors:
                 return tmpl.__of__(self.context)(**kwargs)
             else:
+                if 'graph_updates' in kwargs:
+                    #The response will default to HTML (not JSON) if we try to pass HTML back
+                    self.request.response.setHeader('Content-Type', 'application/JSON;;charset="utf-8"')
```

Comment:
I think you must use `application/json; charset=utf-8` instead:

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type